### PR TITLE
Getting result as Blob

### PR DESCRIPTION
**Pull request by jbghoul**
* Add getBlob and getBlobProxy methods
* Edit documentation
* Add unit tests

### CODE REVIEW

File: doc/proxyfied-requests.md [hunk 1]
```diff
@@ -160,6 +160,19 @@ httpRequest.getRawProxy("http://www.example.com/hello.zip")
     });
 ```
 
+### httpRequest.getBlobProxy
+
+This method can be used to retrieve data as a Blob. It returns a `Blob`.
```
Comment (flozz) [inline doc/proxyfied-requests.md:165]: Like for the other occurrence: add a link to the Blob API:
```suggestion
This method can be used to retrieve data as a Blob. It returns a [`Blob`][blob].
```
+ at the bottom of the document:

```markdown
[blob]: https://developer.mozilla.org/en-US/docs/Web/API/Blob
```

**Review by flozz — CHANGES_REQUESTED** [review]
Everything looks good so far, please just update the doc with the links to the Blob API :)

File: doc/proxyfied-requests.md (hunk 1 above)
Comment (jbghoul) [reply doc/proxyfied-requests.md]: done

File: doc/direct-requests.md [hunk 2]
```diff
@@ -65,6 +65,18 @@ httpRequest.getRaw("http://www.example.com/hello.zip")
     });
 ```
 
+### httpRequest.getBlob
+
+This method can be used to retrieve data as a Blob. It returns a `Blob`.
+```javascript
```
[the remark this answers is not in the document]
Comment (jbghoul) [reply doc/direct-requests.md]: done

**Review by flozz — APPROVED** [review]

**Pull request merged**